README.md

```diff
@@ -51,6 +51,8 @@ provides API endpoints, based on the [OGC API - Features](http://docs.opengeospa
 that enable clients to search for `item`s that match their filtering criteria. The four specifications are meant to be used 
 together, but are designed so each piece is small, self-contained and reusable in other contexts.
 
+**[Overview.md](overview.md)** describes the three core specifications and how they relate to one another.
```

Comment:
A general comment regarding this section: Why is it so focused on the actual file system structure? Why not just write "
```suggestion
The **[Overview](overview.md)** describes the three core specifications and how they relate to one another.
```

And similarly below: 
> The **[Item Specification](item-spec/)** defines ...

> The **[Catalog Specification ](catalog-spec/)** specifies

and finally:

The **[Collection Specification](collection-spec/)** provides

Reply:
Funny, I was wrestling with that when I added it, but was just taking the directory structures as 'given' for some reason, and figuring out how best to fit into that scheme. But you're right, there's no reasons they have to be that.  Will fix it up.

Reply:
Ok, pushed a fix.

Reply:
Great, I removed the appended `/` though. ;-)